components/dashboard/src/teams/TeamSettings.tsx

```diff
@@ -237,22 +221,135 @@ function OrgSettingsForm(props: { org?: OrganizationInfo }) {
-            <TextInputField
-                label="Default Image"
-                // TODO: Provide document links
-                hint="Use any official Gitpod Docker image, or Docker image reference"
-                placeholder={globalDefaultImage}
-                value={defaultWorkspaceImage}
-                onChange={setDefaultWorkspaceImage}
-                disabled={isLoading || !org?.isOwner}
+            <Heading2 className="pt-12">Workspace Images</Heading2>
+            <Subheading className="max-w-2xl">
+                Choose a default image for all workspaces in the organization.
+            </Subheading>
+
+            <WorkspaceImageButton
+                disabled={!org?.isOwner}
+                image={settings?.defaultWorkspaceImage ?? globalDefaultImage ?? ""}
+                onClick={() => setShowImageEditModal(true)}
             />
 
-            {org?.isOwner && (
-                <Button htmlType="submit" className="mt-4" disabled={!org.isOwner}>
-                    Update Default Image
-                </Button>
+            {showImageEditModal && (
+                <OrgDefaultWorkspaceImageModal
+                    settings={settings}
+                    globalDefaultImage={globalDefaultImage}
+                    onClose={() => setShowImageEditModal(false)}
+                />
             )}
         </form>
     );
 }
+
+function WorkspaceImageButton(props: { image: string; onClick: () => void; disabled?: boolean }) {
+    function parseDockerImage(image: string) {
+        // https://docs.docker.com/registry/spec/api/
+        let registry, repository, tag;
+        let parts = image.split("/");
+
+        if (parts.length > 1 && parts[0].includes(".")) {
+            registry = parts.shift();
+        } else {
+            registry = "docker.io";
+        }
+
+        const remaining = parts.join("/");
+        [repository, tag] = remaining.split(":");
+        if (!tag) {
+            tag = "latest";
+        }
+        return {
+            registry,
+            repository,
+            tag,
+        };
+    }
+
+    return (
+        <InputField disabled={props.disabled} className="w-full max-w-lg">
+            <div className="flex items-center bg-gray-50 dark:bg-gray-800 p-3 justify-between rounded-lg">
+                <div className="flex items-center overflow-hidden" title={props.image}>
+                    <span className="w-5 h-5 mr-1">
+                        <Stack />
+                    </span>
+                    <span className="truncate font-medium text-gray-700 dark:text-gray-200">
+                        {parseDockerImage(props.image).repository}
+                    </span>
+                    &nbsp;&middot;&nbsp;
+                    <span className="flex-none w-16 truncate text-gray-500 dark:text-gray-400">
+                        {parseDockerImage(props.image).tag}
+                    </span>
+                </div>
```

Comment:
**suggestion(non-blocking)**: I was thinking if it makes sense to also include who changed the image. This could be part of a future iteration if it makes sense.

| Using default image | Updated image |
|-|-|
| <img width="512" alt="Frame 1835" src="https://github.com/gitpod-io/gitpod/assets/120486/7e48011a-b1c4-4626-91fc-05a5fe7216de"> | <img width="512" alt="Frame 1834" src="https://github.com/gitpod-io/gitpod/assets/120486/647ea078-ed56-4715-9c6c-12167f5ad281"> |